### ISSUE TITLE
Obsoletion Term merge: RNA polymerase I core factor complex (GO:0070860), RNA polymerase transcription factor SL1 complex (GO:0005668)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#22214]
Yeast has
RNA polymerase I core factor complex (GO:0070860)
A RNA polymerase I-specific transcription factor complex that is required for the transcription of rDNA by RNA polymerase I. In yeast the complex consists of Rrn6p, Rrn7p, and Rrn11p. [PMID:8702872]
Rrn6 =     ?
Rrrn7 = 	TAF1B
Rrn11 =    ?

Human has
RNA polymerase transcription factor SL1 complex (GO:0005668)
A RNA polymerase I-specific transcription factor complex that contains the TATA-box-binding protein (TBP) and at least three TBP-associated factors including proteins known in mammals as TAFI110, TAFI63 and TAFI48. [PMID:15691654]
TAFI110  TAF1C ?? Looks like rrn6 (AlphaFold)
TAFI63	TAF1B
TAFI48	TAF1A ?? Looks like rrn11. (AlphaFold) 

* **Reference(s)**

Googling "TAF1C and rrn6" found
https://pubmed.ncbi.nlm.nih.gov/28340337/
Which confirms these orthologies.
* **Any other information**

I mailed Pfam to

https://pfam.xfam.org/family/PF10214 
To include the human members in
https://www.ebi.ac.uk/interpro/entry/panther/PTHR15319/ 

And
https://pfam.xfam.org/family/PF04090
https://pfam.xfam.org/family/PF14929

**Comment by ValWood** [comment]
spotted tghrough a PAINT Mapping for pombe to 
RNA polymerase transcription factor SL1 complex (GO:0005668)
for rrn6

**Comment by ValWood** [comment]
RNA polymerase transcription factor SL1 complex (GO:0005668)
has ony 8 EXP. 
@pgaudet  do you agree to obsolete and merge into GO:0070860   [ JSON](https://www.ebi.ac.uk/QuickGO/services/ontology/go/terms/GO:0070860/complete)
RNA polymerase I core factor complex

**Comment by ValWood** [comment]
can I obsolete 
RNA polymerase transcription factor SL1 complex (GO:0005668)
replace by 
RNA polymerase I core factor complex
add exact synonym?

**Comment by ValWood** [comment]
in yeast core factor is composed of
Core Factor (CF) composed of subunits Rrn6, Rrn7, and Rrn11
The CF subunits assemble through an interconnected network of interactions between five structural domains that are conserved in orthologous subunits of the human Pol I factor SL1. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4219626/

It seems the names are  not shared at all, but we could make
RNA polymerase I core factor/SL1 complex
as a portmanteau term to group these orthologous, functionally equivalent complexes?

**Comment by pgaudet** [comment]
I see EXP annotations by FlyBase and MGI; maybe @hattrill and @LiNiMGI have feedback? 
https://amigo.geneontology.org/amigo/term/GO:0005668

**Comment by hattrill** [comment]
I am fine with this proposal. Agree that modifying name to reflect those in use is a good idea.

**Comment by LiNiMGI** [comment]
@krchristie Would you please take a quick look of this one? Thanks!

**Comment by ValWood** [comment]
Actually looking back to the beginning of this ticket, the ortholog is unconfirmed. I meant to ask Pfam to see if these families  should extend/merge. I'll do this first.
There is quite a bit of literature saying the complexes are functionally equivalent, but it would make more sense if the ortholog was confirmed first.

**Comment by ValWood** [comment]
I see I have already mailed Pfam, will check for response.

**Comment by ValWood** [comment]
This is a summary of the connections

I will also submit to Panther:

yeast rrn6 = human TAF1C

InterPro/Pfam

The domain 
IPR049087 TAF1C, beta-propeller domain (PF20641)
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR049087/

Is
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR048535/
IPR048535 RRN6, beta-propeller PF10214

And the domain
TAF1C, helical bundle domain (PF20642)
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR049090/

Is
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR048536/
IPR048536 RRN6, K-rich C-terminal domain (PF20639)

PANTHER 
Yeast https://www.pantherdb.org/panther/family.do?clsAccession=PTHR28221
Is
Metazoan https://www.pantherdb.org/panther/family.do?clsAccession=PTHR15319

The connection is documented in the literature but isn't captured by any ortholog predictor or family database

======


Yeast Rrn11 = humans TAF1A
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR039495/

IPR039495
TATA box-binding protein-associated factor RNA polymerase I subunit A (PF14929)
https://www.ebi.ac.uk/interpro/entry/InterPro/IPR039495/

PANTHER 

Yeast https://www.pantherdb.org/panther/family.do?clsAccession=PTHR28244
Is
Metazoan https://www.pantherdb.org/panther/family.do?clsAccession=PTHR32122

**Comment by krchristie** [comment]
Based on the reference that @ValWood found via Googling "TAF1C and rrn6" (citation and relevant quote below), I agree that having one term to represent the complexes currently represented by the two separate terms "**RNA polymerase I core factor complex (GO:0070860)**" & "**RNA polymerase transcription factor SL1 complex (GO:0005668)**".

Engel C, Gubbey T, Neyer S, Sainsbury S, Oberthuer C, Baejen C, Bernecky C, Cramer P. **Structural Basis of RNA Polymerase I Transcription Initiation.** Cell. 2017 Mar 23;169(1):120-131.e22. doi: 10.1016/j.cell.2017.03.003. PMID:28340337.

> The human counterpart of CF, selectivity factor (SL) 1 (Comai et al., 1992; Learned et al., 1985), comprises homologs to **Rrn6 (TAF1C)**, **Rrn7 (TAF1B)**, **Rrn11 (TAF1A)** (Russell and Zomerdijk, 2006), and the additional subunits TAF1D and TAF12 

To summarize the above sentence diagrammatically, you get this:

**Scer   =  human**
Rrn6  = TAF1C
Rrrn7 = TAF1B
Rrn11 = TAF1A
 none    = TAF1D
  none    = TAF12

**Comment by ValWood** [comment]
I can't detect TAF1D outside chordata, it's low complexity and has no identifiable domains.
TAF12 = yeast Taf12. This is a member of SAGA /TFIID  (may or may not be a bona fida member of SL1).

I feel comfortable  representing  RNA polymerase I core factor/SL1 complex as a single complex.
Will go ahead.

**Comment by ValWood** [comment]
I just spotted a comment on 
GO:0070860   RNA polymerase I core factor complex
Note that, although this complex can be considered analogous to the mammalian transcription factor SL complex, the core factor complex does not include TBP, whereas SL1 does.

so these complexes are defined differently.
closing
